doc/tutorials/intermediate/structure_data_store.md

```diff
@@ -1,15 +1,14 @@
 # Structure with a DataStore
```

Comment:
This application does not seem very robust. As soon as I drag one of the range slider the chart does not show anything.

![image](https://github.com/holoviz/panel/assets/42288570/3169c0e3-06dd-4012-be28-ebc57d530772)

And moving the range slider back in place does not display anything in the plot again.